libs/adapters/src/daemon.ts

```diff
@@ -1,7 +1,9 @@
 import { CacheNamespaces } from '@hicommonwealth/core';
 import { logger } from '@hicommonwealth/logging';
+import { fileURLToPath } from 'url';
 import { CacheDecorator, KeyFunction } from './redis';
 
+const __filename = fileURLToPath(import.meta.url);
```

Comment:
Filepath could be fine in production since we use structured logs but it would be really ugly for local dev since the file name is a log prefix used for pretty-printing e.g. `[16:30:37.390] INFO (globalActivityCache.ts): ....`. I would say it's better to leave as is and log the filepath.